examples/jsm/webxr/XRControllerModelLoader.d.ts

```diff
@@ -0,0 +1,24 @@
+import {
+	Group,
+	Object3D,
+	Texture
+} from '../../../src/Three';
+
+import { GLTFLoader } from '../loaders/GLTFLoader';
+
+export class XRControllerModel extends Object3D {
+	constructor( );
+
+	motionController: any;
+
+	setEnvironmentMap( envMap: Texture ): XRControllerModel;
+}
+
+export class XRControllerModelLoader {
+	constructor( gltfLoader?: GLTFLoader );
```

Comment:
At some point we may want to expand the @webxr-input-profiles/assets schema to support more than just gltf/glb so let's maybe change this to be passed in a generic Loader?